scripts/start.sh

```diff
@@ -55,7 +55,7 @@ wait_for_pods() {
 print_hosts() {
   retry_command "ingress" "${KUBECTL} get ingress -o json \
     | jq --exit-status '.items[0].status.loadBalancer.ingress'"
-  ${KUBECTL} get ingress --no-headers | awk '{print $3 " " $2}'
+  ${KUBECTL} get ingress --no-headers | awk -v ip=$(minikube ip) '{print ip " " $2}'
```

Comment:
This puts a hard dependency on minikube which I'd rather avoid.

Reply:
True, but only for the 'print-hosts' part...which should (mostly) only be needed when running locally via minikube.